languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -102505,6 +102505,49 @@ The accident victim died from her injuries.
             <example correction="">This <marker>is</marker> right.</example>
             <example>This <marker>is</marker> conforming with my current knowledge.</example>
         </rule>
+        <rule id="TELL_IT" name="I [told it to] -> I [told] Leo" default="temp_off">
+            <antipattern>
+                <token>to</token>
+                <token>tell</token>
+            </antipattern>
+            <antipattern>
+                <token>tell</token>
+                <token skip="-1">it</token>
+                <token regexp="yes">it|straight</token>
+                <token postag="SENT_END" />
+            </antipattern>
+            <antipattern>
+                <token inflected="yes">tell</token>
+                <token>it</token>
+                <token>to</token>
+                <token postag="PRP$" />
+                <token>face</token>
+            </antipattern>
+            <antipattern>
+                <token inflected="yes">tell</token>
+                <token>it</token>
+                <token>to</token>
+                <token>Groucho</token>
+            </antipattern>
+            <pattern>
+                <marker>
+                    <token inflected="yes">tell</token>
+                    <token>it</token>
+                    <token>to</token>
+                </marker>
+            <token chunk_re="B\-NP\-(singular|plural)">
```

Comment:
no need to escape the `-` here, and since there are only singular and plural, you could write `B-NP.*`